### DISCUSSION TITLE
Is there a way to show the original image instead of a rendered?

### DISCUSSION THREAD

**Discussion by arsleathelas** [ValentinH/react-easy-crop#322] [Q&A]
Hello, I'm wondering if there is a way to show the original intrinsic size of the image that we are working on instead of a rendered one?

**Comment by ValentinH** [comment]
I'm not sure to fully understand what you have in mind. Could you please detail a bit more? 🙂

**Reply by arsleathelas** [reply]
yes, when I inspect the image of the dog in one of the examples... developer tools shows the image being rendered, so we are not really working on the original image resolution, is there a way to not render the original image?
![ArcoLinux_2021-12-03_17-53-46](https://user-images.githubusercontent.com/95470278/144641812-7d537b1e-5248-44a0-a3e5-85fcf79caf7b.png)

**Reply by ValentinH** [reply]
This is not linked to this library but just how the img element works.if you want to render it to it's full size, you need to make the container 1910x1000px for this image.

Why do you worry about?
